tools/firtool/firtool.cpp

```diff
@@ -322,6 +322,22 @@ static cl::opt<bool> etcDisableModuleInlining(
     cl::desc("Disable inlining modules that only feed test code"),
     cl::init(false), cl::cat(mainCategory));
 
+enum class RandomKind { None, Mem, Reg, All };
+
+static cl::opt<RandomKind> disableRandom(
+    cl::desc("Disable random initialization code (may break semantics!)"),
+    cl::values(clEnumValN(RandomKind::Mem, "disable-mem-randomization",
+                          "Disable emission of memory randomization code"),
+               clEnumValN(RandomKind::Reg, "disable-reg-randomization",
+                          "Disable emission of register randomization code"),
+               clEnumValN(RandomKind::All, "disable-all-randomization",
+                          "Disable emission of register randomization code")),
```

Comment:
This should indicate it disables all randomization.

Reply:
Whoops, fixed.  Thanks!